### ISSUE TITLE
Map Themes control

### ISSUE THREAD

**Issue by jokd** [origo-map/origo#1192]
**Is your feature request related to a problem? Please describe.**
It would be nice to be able to switch between "map themes" with the click of a button. Similar to how the bookmarks control works but instead of moving in the map you alter the map config (or maybe just the groups and layers). This way you can have different configurations for different tasks / users and easily switch between them.

**Comment by Grammostola** [comment]
I had a discussion yesterday with my colleagues about something similar. Presently we employ a simple "map switcher" plugin which is a dropdown under the main menu that shows names and icons for differently themed map applications and loads those separate applications when a selection is made and feedback has been received that it isn't too clear which map one is currently employing, which led the discussion toward moving the dropdown from the menu so that the current map name / icon could always be visible to that it would be nice if the map selector could be one of the round buttons showing just a pretty icon for that map and when clicked/ pushed would show a selector of some sort. My thought is that it would be neat if the current application / address didn't change but rather just the groups and layers, as you mention; my supposition is that your suggestion would also employ an indication of the currently selected theme.

**Comment by jokd** [comment]
Yes, nice buttons we like. Guess we can discuss this further when we start digging in this.
